book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -192,13 +185,9 @@ game.run();
 # }
 ```
 
-Here we're creating a new `WindowBundle` that uses the config we prepared above.
-That bundle is being used as a part of `GameDataBuilder`, a central repository
-of all the game logic that runs periodically during the game runtime.
-
-> **Note:** We will cover systems and bundles in more details later, for now, think of the
-> bundle as a group of functionality that together provides a certain feature to the engine.
-> You will surely be writing your own bundles for your own game's features soon.
+Here we're creating a new instance of `GameDataBuilder`, a central repository
+of all the game logic that runs periodically during the game runtime. Right now it's empty,
+but soon we will start adding there all sorts of systems and bundles that run our game code.
```

Comment:
`Right now it's empty, but soon we willst start adding all sorts of systems and bundles to it - which will run our game code.`